### ISSUE TITLE
Decouple RDS and S3 installation

### ISSUE THREAD

**Issue by rrrkharse** [awslabs/kubeflow-manifests#22]
Provide customers with the flexibility to choose which AWS service(s) they want to integrate with

**Comment by Harikantipudi** [comment]
Yes I have been looking for this since I would not want to go to S3 yet

**Comment by surajkota** [comment]
@Harikantipudi this feature is available in 1.4 and 1.5. https://awslabs.github.io/kubeflow-manifests/docs/deployment/rds-s3/guide/#using-only-rds-or-only-s3
